cmd/cluster_phase_install_masters.go

```diff
@@ -0,0 +1,67 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/xetys/hetzner-kube/cmd/phases"
+	"github.com/xetys/hetzner-kube/pkg"
+	"github.com/xetys/hetzner-kube/pkg/clustermanager"
+	"github.com/xetys/hetzner-kube/pkg/hetzner"
+)
+
+func init() {
+	command := declarePhaseCommand("install-masters", "install the control plane", func(cmd *cobra.Command, args []string) {
+		clusterName := args[0]
+		keepCa, _ := cmd.Flags().GetBool("keep-ca")
+		keepAll, _ := cmd.Flags().GetBool("keep-all-certs")
+		phaseOptions := phases.InstallMastersPhaseOptions{
+			KeepCaCerts:  keepCa,
+			KeepAllCerts: keepAll,
+		}
+
+		_, cluster := AppConf.Config.FindClusterByName(clusterName)
+		provider := hetzner.NewHetznerProvider(AppConf.Context, AppConf.Client, *cluster, AppConf.CurrentContext.Token)
+		masterNode, err := provider.GetMasterNode()
+		FatalOnError(err)
+		err = AppConf.SSHClient.(*clustermanager.SSHCommunicator).CapturePassphrase(masterNode.SSHKeyName)
+		FatalOnError(err)
+		coordinator := pkg.NewProgressCoordinator()
+
+		for _, node := range provider.GetAllNodes() {
+			steps := 3
```

Comment:
IMHO we make the code more comprehensible if we do no use code like `step := 3` that hide what we are supposing to do here. Maybe we can iterate over step, add a step name and check name here? Or we can use a chained list on step definition so we can know what's the next step?

Reply:
The issue with the steps here is, that it's currently a pity with the UIProgress. If you once ran over 100% before it is actually done, it sends a `done` signal to the channel and the bar synchronization is not really working. This leads to the progress always showing some work, as it actually is already complete. You must know the steps before.

I'm also considering a different way of showing progress. Initially, I liked the UI Progress, but it is so ugly in handling. WDYT?